with-legend-state-supabase/.gitignore

```diff
@@ -0,0 +1,17 @@
+node_modules/
```

Comment:
As per our writing contribution guide, we prefer not to include gitignore files within an example directory: https://github.com/expo/examples/blob/master/contributing.md

Can you please remove this?